saleor/graphql/channel/mutations/channel_create.py

```diff
@@ -165,6 +165,28 @@ class PaymentSettingsInput(BaseInputObjectType):
             "requested action for the transaction." + ADDED_IN_316 + PREVIEW_FEATURE
         ),
     )
+    release_funds_for_expired_checkouts = graphene.Boolean(
+        required=False,
+        description=(
+            "Determine if the funds for expired checkouts should be released automatically."
+            + ADDED_IN_320
+        ),
+    )
+    checkout_ttl_before_releasing_funds = Hour(
+        required=False,
+        description=(
+            "The time in hours after which funds for expired checkouts will be released."
+            + ADDED_IN_320
+        ),
+    )
+    checkout_release_funds_cut_off_date = DateTime(
+        required=False,
+        description=(
+            "The date after which the funds for expired checkouts will be released."
```

Comment:
Not. sure, if I understand the description. 
Based on the name, I assume that this is some kind of the limiter for not touching the old transactions. Right? 
In that case, why it is DateTime? Shouldn't be a delta?

Reply:
But the description is still hard to understand. 
Maybe something like:

It specifies the earliest date on which funds for expired checkouts can begin to be released. Expired checkouts dated before this cut-off will not have their funds released. Additionally, no funds will be released for checkouts that are more than one year old, regardless of the cut-off date.

Reply:
The `one year old` needs to be also clarified.  It could be one year since creation date or one year since last change. 
In the code, I see that we use last_change, but imo this needs to be said explicitly